1_wqpdata/src/wqp_inventory.R

```diff
@@ -85,3 +82,59 @@ inventory_wqp <- function(ind_file, wqp_state_codes, wqp_states, wqp_codes) {
 
   invisible()
 }
+
+# prepare a plan for getting an inventory of available WQP data
+plan_inventory <- function(constituents, folders) {
+
+  inventory <- scipiper::create_task_step(
+    step_name = 'inventory',
+    target_name = function(task_name, step_name, ...) {
+      scipiper::as_ind_file(file.path(folders$tmp, sprintf('inventory_%s.feather', task_name)))
+    },
+    command = function(task_name, ...) {
+      sprintf("inventory_wqp(target_name, wqp_state_codes, wqp_states, wqp_codes, I('%s'))", task_name)
+    }
+  )
+
+  # steps: download, post
```

Comment:
the downloading from WQP and posting to Drive are both happening in `inventory_wqp()` above, so this step is probably just downloading [from Drive, for people who didn't do the initial WQP on their own computer]